engine/lib/views.php

```diff
@@ -1519,6 +1519,9 @@ function elgg_views_boot() {
 
 	elgg_register_js('elgg', elgg_get_simplecache_url('elgg.js'), 'head');
 	elgg_load_js('elgg');
+	
+	elgg_register_css('fontawesome', 'vendor/fortawesome/font-awesome/css/font-awesome.min.css');
```

Comment:
Don't point to the vendor file directly. Instead, register it as a view with `engine/views.php` and then point to the simplecache asset. View should be "font-awesome.css"